js/views/components/CryptoCurSelector.js

```diff
@@ -134,6 +139,7 @@ export default class extends baseVw {
       // If active or disabled lists are passed in, we'll assume they're
       // different and ensure the processedCurrencies list reflects them.
       processedState.processedCurs = processedState.processedCurs.map(cur => ({
+        ...cur,
```

Comment:
I caught a bug here, if the currencies weren't recreated the values of each one were being set to just active and disabled.

Reply:
yeah, good catch.